mimic/model/glance_objects.py

```diff
@@ -0,0 +1,102 @@
+"""
```

Comment:
I was hoping we could use the same class for both the server_image and glance_images as they are essentially the same. Except for the formatting like all of the metadata in the server_images are flattened on the image object.